Lib/test/test_time.py

```diff
@@ -159,6 +159,13 @@ def test_sleep(self):
         self.assertRaises(ValueError, time.sleep, -1)
         time.sleep(1.2)
 
+    def test_epoch(self):
+        # bpo-43869: Make sure that Python use the same Epoch on all platforms:
+        # January 1, 1970, 00:00:00 (UTC).
+        epoch = time.gmtime(0)
+        # Only test the date and time, ignore other gmtime() members
```

Comment:
We could test more members, but I don't think that it is worth it and it can make the test more fragile if the exact value changes depending on the platform.